wcvendors/wcvendors
# Added: Make commissions table in WP Admin filterable

### Changes proposed in this Pull Request:

This adds filters to the commission's table so that 3rd party developers can hook into this table. 

Closes #715.

### How to test the changes in this Pull Request:

1. Use the following filters to test the hooks
2. See new col and data 

`//  Add the column 
add_filter( 'wcvendors_commissions_columns', 'test_col' );
function test_col( $columns ){ 
	$columns[ 'testcol'] = 'Test Column';
	return $columns;
}

// Hook into the column value 
add_filter( 'wcvendors_commissions_column_default_testcol', 'populate_testcol', 10, 3 ); 
function populate_testcol( $value, $item, $column_name ){ 
	// Show the commission internal ID for testing
	$value = $item->id; 
	return $value; 
}`
